scripts/predict.py

```diff
@@ -10,36 +11,48 @@
 
 @click.command()
 @click.option('--mask_path', required=True,
-              help='Adress of json file of the mask')
+              help='Path of json file of the mask')
 @click.option('--picture_path', required=True,
-              help='Adress of picture to be predicted')
+              help='Path of picture to be predicted')
 @click.option('--model_path', required=True,
```

Comment:
@Adman I don't really see the value of `required=True` if the script can not run without it should in my opinion be a argument of the script rather than option. This way we are forcing a user to type more than necessary but that is just my opinion feel free to disregard it if you don't agree but I would like to also maybe hear @mrshu take on this. Also I think the standard of using arguments/option is that if it is specified in a bash script or somewhere people usually use longer form of the argument because it is more descriptive and helps with reading the script but when running from command line it they want to use abbreviations of these arguments because it is faster and less typing so I think every argument/option should have a shorter form.